### PR TITLE
wip: silverstripe 5 compatibility 

### PR DESCRIPTION
**Pull request by tractorcow**
This is just a preview PR to be replaced once the current WIP is ready to merge.

### CODE REVIEW

File: client/src/styles/fluent.scss [hunk 1]
```diff
@@ -156,24 +157,24 @@ $fluent-color-localised: rgb(102, 15, 242);
...
+  font-size: 1em;
 }
 
 // Gridfield menu items
-.action-group-COPY_FROM .action-menu__toggle:before {
+.action-group-COPY_FROM .action-menu__toggle:before {// sass-lint:disable-line class-name-format
   // force ... to turn into actual title
   // @todo i18n this
-  content: "Copy from" !important;
+  content: "Copy from" !important;// sass-lint:disable-line no-important
```
Comment (tractorcow) [inline client/src/styles/fluent.scss:171]: This was a really ugly work-around because I couldn't update the button labels in react menu.

Comment (GuySartorelli) [reply client/src/styles/fluent.scss:171]: Probably worth adding that as a comment above it - it's always nice when looking at scss which has exceptions to see comments right there telling you _why_ it's an exception.

Comment (tractorcow) [reply client/src/styles/fluent.scss:171]: Yeah it was '...' which is the default label for 'more actions'.

Comment (GuySartorelli) [reply client/src/styles/fluent.scss:171]: Ahh that makes sense haha ignore me then, I just didn't understand that comment that was already there 😅

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "tractorcow/silverstripe-fluent",
+    "name": "silverstripe/silverstripe-fluent",
```
Comment (maxime-rainville) [inline composer.json:2]: That was only added here so our fork could be installed under its own name.

```suggestion
    "name": "tractorcow/silverstripe-fluent",
```

File: readme.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# WARNING
+
+This is a temporary fork of `tractorcow/silverstripe-fluent`. Do not install in production environment.
```
Comment (maxime-rainville) [inline readme.md:3]:
```suggestion
```

File: src/Control/LocaleAdmin.php [hunk 4]
```diff
@@ -34,8 +34,8 @@ class LocaleAdmin extends ModelAdmin
     protected function init()
     {
         parent::init();
-        Requirements::javascript('tractorcow/silverstripe-fluent:client/dist/js/fluent.js');
-        Requirements::css("tractorcow/silverstripe-fluent:client/dist/styles/fluent.css");
+        Requirements::javascript('silverstripe/silverstripe-fluent:client/dist/js/fluent.js');
+        Requirements::css("silverstripe/silverstripe-fluent:client/dist/styles/fluent.css");
```
Comment (maxime-rainville) [inline src/Control/LocaleAdmin.php:38]:
```suggestion
        Requirements::javascript('tractorcow/silverstripe-fluent:client/dist/js/fluent.js');
        Requirements::css("tractorcow/silverstripe-fluent:client/dist/styles/fluent.css");
```

File: src/Extension/FluentLeftAndMainExtension.php [hunk 5]
```diff
@@ -23,8 +23,8 @@ class FluentLeftAndMainExtension extends Extension
 
     public function init()
     {
-        Requirements::javascript("tractorcow/silverstripe-fluent:client/dist/js/fluent.js");
-        Requirements::css("tractorcow/silverstripe-fluent:client/dist/styles/fluent.css");
+        Requirements::javascript("silverstripe/silverstripe-fluent:client/dist/js/fluent.js");
+        Requirements::css("silverstripe/silverstripe-fluent:client/dist/styles/fluent.css");
```
Comment (maxime-rainville) [inline src/Extension/FluentLeftAndMainExtension.php:27]:
```suggestion
        Requirements::javascript("tractorcow/silverstripe-fluent:client/dist/js/fluent.js");
        Requirements::css("tractorcow/silverstripe-fluent:client/dist/styles/fluent.css");
```

**Comment by GuySartorelli** [comment]
A lot of these commits are already in this repo - they should be merged up to master before this pr gets merged.

**Comment by tractorcow** [comment]
Do you want me to change the source from `silverstripe:7` to `silverstripe:master`

**Comment by GuySartorelli** [comment]
No, `silverstripe:7` is the correct source. It just has some commits from this repo which aren't in this repo's master branch (I'm guessing when the branch was made it was branched off `6` or `6.0` instead of master, but I'm not sure)

**Comment by GuySartorelli** [comment]
@tractorcow if you can merge up the 6 branch to master (see https://github.com/tractorcow-farm/silverstripe-fluent/compare/master...6) then I'll update this PR with the recommended changes and we'll be able to get it merged.

**Comment by tractorcow** [comment]
I'll do this soon. I will make a 7 branch once we tag our first 7.0.0

**Comment by tractorcow** [comment]
Completed.

**Comment by GuySartorelli** [comment]
Sweet, I'll update the PR in a moment

**Comment by GuySartorelli** [comment]
I've opted to open a new PR, since I can't amend this one and didn't want to affect our current CI by making changes to the fork used in the beta
https://github.com/tractorcow-farm/silverstripe-fluent/pull/779

**Comment by tractorcow** [comment]
Replaced with https://github.com/tractorcow-farm/silverstripe-fluent/pull/779

**Pull request closed without merging**